### PR TITLE
using ugi instead of connection 

### PR DESCRIPTION
**Pull request by lalaguozhe**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -132,6 +132,16 @@
...
+			<artifactId>hive-cli</artifactId>
+			<version>0.8.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.hive</groupId>
+			<artifactId>hive-builtins</artifactId>
+			<version>0.8.1</version>
+		</dependency>
+		<dependency>
 			<groupId>org.apache.hadoop</groupId>
```
Comment (yixzhang) [inline pom.xml:145]: these 2 dependencies are not needed anymore

File: src/main/java/com/dianping/cosmos/hive/server/FileDownload.java [hunk 2]
```diff
@@ -50,12 +51,14 @@ protected void doGet(HttpServletRequest request,
 			if (output != null)
 				try {
 					output.close();
-				} catch (IOException ignore) {
+				} catch (IOException ioe) {
+					ioe.printStackTrace();
```
Comment (yixzhang) [inline src/main/java/com/dianping/cosmos/hive/server/FileDownload.java]: print it to log4j

File: src/main/java/com/dianping/cosmos/hive/server/FileDownload.java [hunk 3]
```diff
@@ -50,12 +51,14 @@ protected void doGet(HttpServletRequest request,
...
 				}
 			if (input != null)
 				try {
 					input.close();
-				} catch (IOException ignore) {
+				} catch (IOException ioe) {
+					ioe.printStackTrace();
```
Comment (yixzhang) [inline src/main/java/com/dianping/cosmos/hive/server/FileDownload.java]: print it to log4j

File: src/main/java/com/dianping/cosmos/hive/server/queryengine/jdbc/HiveJdbcClient.java [hunk 4]
```diff
@@ -280,8 +357,45 @@ public HiveQueryOutput getQueryResult(String tokenid, String username,
...
+						
+						c = DriverManager.getConnection(
+								Krb5Login.HIVE_CONNECTION_URL, "", "");
+						
+						if (logger.isDebugEnabled()) {
+							logger.debug("through get connection ugi username:" + ugi.getUserName()) ;
+						}
+					} catch (Exception e) {
+						logger.error("get connection failed：" + e.getMessage());
+						e.printStackTrace();
```
Comment (yixzhang) [inline src/main/java/com/dianping/cosmos/hive/server/queryengine/jdbc/HiveJdbcClient.java]: remove it

File: src/main/resources/hive-site.xml [hunk 5]
```diff
@@ -1,13 +1,1495 @@
 <?xml version="1.0"?>
 <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+	license agreements. See the NOTICE file distributed with this work for additional 
+	information regarding copyright ownership. The ASF licenses this file to 
+	You under the Apache License, Version 2.0 (the "License"); you may not use 
+	this file except in compliance with the License. You may obtain a copy of
```
Comment (yixzhang) [inline src/main/resources/hive-site.xml:7]: please pickup the necessary ones

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -2,22 +2,24 @@
...
-				<h1>Hive Web Client,Helping User For On-Demand Query</h1>
+				<h1>Hive Web Client, Helping User For Adhoc Query</h1>
 				<ul>
 					<li><a href="#">Home</a></li>
 					<li><a href="#">Hive Query</a></li>
 					<li><a href="#">Table Schema</a></li>
 					<li><a href="#">Query History</a></li>
-					<li><a href="http://10.1.1.172:50030/jobtracker.jsp" target="_blank">Job Tracker</a></li>
+					<li><a href="http://10.2.6.101:50030/jobtracker.jsp" target="_blank">Job Tracker</a></li>
 				</ul>
```
Comment (yixzhang) [inline src/main/webapp/index.html:23]: put the URL in the pom

File: src/main/java/com/dianping/cosmos/hive/server/queryengine/jdbc/HiveJdbcClient.java (hunk 4 above)
Comment (lalaguozhe) [inline src/main/java/com/dianping/cosmos/hive/server/queryengine/jdbc/HiveJdbcClient.java]: 这个是需要的，通过这个获取一个connection

**Pull request merged**